CMakeLists.txt

```diff
@@ -48,9 +48,7 @@ set_package_properties(Boost PROPERTIES
     URL "https://www.boost.org"
     PURPOSE "Used throughout OMPL for data serialization, graphs, etc.")
 set(Boost_USE_MULTITHREADED ON)
-set(Boost_NO_BOOST_CMAKE ON)
```

Comment:
I cannot tell what reason a user would enable this. CMake developers recommend the use of CONFIG mode, whenever possible, but this disables it.

Reply:
This was probably put in when the Boost CMake support was still experimental. I'm in favor of this change.